pinot-common/src/main/java/org/apache/pinot/common/config/TableConfig.java

```diff
@@ -354,6 +387,7 @@ public ZNRecord toZNRecord()
     }
 
     ZNRecord znRecord = new ZNRecord(_tableName);
+    simpleFields.put(UPDATE_SEMANTIC_CONFIG_KEY, _updateSemantic.toString());
```

Comment:
This can cause NPE

Reply:
updated to follow the convention of rest of the code